TLM/TLM/State/OptionsTabs/MaintenanceTab_ToolsGroup.cs

```diff
@@ -55,5 +61,20 @@ private static void DoRemoveTrafficLights()
 
         private static void OnResetSpeedLimitsClicked()
             => SpeedLimitManager.Instance.ResetSpeedLimits();
+
+        private static void OnResetLaneConnectionsClicked() {
+            ConfirmPanel.ShowModal(
+                T("Maintenance.Dialog.Title:Remove all lane connections"),
+                T("Maintenance.Dialog.Text:Remove all lane connections, Confirmation"),
+                (_, result) => {
+                    if (result == 1) {
+                        SimulationManager.instance.AddAction(
+                            () => {
+                                LaneConnectionManager.Instance.RemoveAllLaneConnections();
+                                OptionsManager.UpdateRoutingManager();
```

Comment:
This call to update routing manager does not belong to UI handler, do it in the RemoveAllLaneConnections or something like that.

Reply:
Yeah, good point 👍